arrow/locales.py

```diff
@@ -2832,7 +2832,7 @@ class HindiLocale(Locale):
     month_abbreviations = [
         "",
         "जन",
-        "फ़र",
+        "फ़र",
```

Comment:
I edited the file through the GitHub page, so the browser may have refilled some characters